submissions/t-jay-777/Hooli-style Popup/index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Hooli-style Popup</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+       <div class="container">
+        <div class="nav">
+            <ul class="nav-menu">
+                <li class="nav-submenu">
+                    <a href="#">Mail</a>
+                </li>
+                <li class="nav-submenu">
+                    <a href="#">Images</a>
+                </li>
+                <li class="nav-submenu popup">
+                    <a href="#popup-menu"><img src="img/apps.jpg" alt="apps"></a>
+                    <div class="popup-menu-1">
+						<div class="popup-menu" id="popup-menu">
```

Comment:
Use Prettier for auto-formatting your code.

Reply:
I'll try to use it.